### DISCUSSION TITLE
Strange behavior after installation on new computer.

### DISCUSSION THREAD

**Discussion by mutti-wilson** [GrandOrgue/grandorgue#2068] [Help]
My previous computer running GO had a motherboard issue so I've been resetting up with a new PC. Everything is more or less set up but I'm not able to right click on most of the midi objects on Friesach to assign them. Also, when I close the main Friesach panel window, the entire program crashes. I'm running the latest build.

**Comment by midimusic** [comment]
This is obviously a computer problem so if you need help you must tell us what computer and operating system you are using!!!
csw900

**Reply by mutti-wilson** [reply]
i7-6700T 3.6GHz, 32GB RAM, 1TB NVMe SSD. Audio interface is a Behringer UMC204HD. Windows 11 Pro.

I'll check with another organ and see if it works.

**Reply by mutti-wilson** [reply]
Same behavior on Giubiasco. Multiple midi elements do not appear. I cannot assign the pedals either by using the main organ screen or through the midi objects window. When I click on configure through the midi objects it bounces to the main screen with no dialog box.

**Reply by midimusic** [reply]
It is almost certain that your compiuter has a hardware fault, probably memory.
You need to do a thorough memory check. Windows 10 had a built in memory checker, if Windows 11 also has a memory checker then use that. You need to check ALL of the memory using a checker that does the checks after booting and before loading windows. You can ask co-pilot how to test the memory.
csw900

**Reply by mutti-wilson** [reply]
Just ran the windows memory test and it passed with no errors. I consider myself fairly computer savvy and this has me stumped.

**Reply by mutti-wilson** [reply]
I use 2 touchscreens and I swapped the main screen from one to the other in the display settings and everything works now. I don't know why and don't care but it's all working good now!
